docroot/profiles/custom/cgov_site/modules/custom/cgov_vocab_manager/src/Form/CgovVocabManagerForm.php

```diff
@@ -209,6 +209,7 @@ public function buildForm(array $form, FormStateInterface $form_state, Vocabular
         continue;
       }
       // Count entries after the current page.
+      /* @phpstan-ignore-next-line */
```

Comment:
$complete_tree is not defined before this line, thats why it is throwing phpstan warning.

Reply:
Resolved